eng/pipelines/runtimelab.yml

```diff
@@ -81,9 +62,9 @@ stages:
       buildConfig: Release
       platforms:
       - Linux_x64
```

Comment:
Why are we keeping the Linux job?

Reply:
We don't have to. For now I wanted to have it as baseline for comparisons and A/B testing (if needed)